src/types.js

```diff
@@ -6,21 +6,38 @@ export type QuiqChatSettings = {
   BURNED?: boolean,
 };
 
-export type EventType = 'Text' | 'Join' | 'Leave';
+export type EventType = 'Join' | 'Leave' | 'Register';
 export type AuthorType = 'Customer' | 'Agent';
-export type MessageType = 'Text' | 'ChatMessage' | 'BurnItDown';
+export type TextMessageType = 'Text';
```

Comment:
Nitpick, but it's probably clearer to just use the string literal instead of using a type alias for it